### PR TITLE
Add jdbc and flyway dependencies

### LINKED ISSUE #10: Add FlyWay

### PR DESCRIPTION
**Pull request by SabuhiYusif**
This PR fixes many things:

- Adds flyway, jdbc, postgres dependencies
- Connects jdbc to postgres
- Changes db name from `tasker` to `taskerdb`
- Changes application properties to yaml file
- Changes port for postgres from default(5432) to 5555 in order to avoid port conflicts with local databases

also fixes #10

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -15,6 +15,9 @@ repositories {
 
 dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation("org.springframework.boot:spring-boot-starter-jdbc")
```
Comment (javidy) [inline build.gradle:18]: maybe change it to `implementation 'org.springframework.boot:spring-boot-starter-jdbc'` to be consistent with other dependency declarations

File: build.gradle [hunk 2]
```diff
@@ -15,6 +15,9 @@ repositories {
...
+    implementation("org.flywaydb:flyway-core")
+    runtimeOnly("org.postgresql:postgresql")
```
Comment (javidy) [inline build.gradle:20]: what is it for?

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+spring:
```
Comment (javidy) [inline src/main/resources/application.yml:1]: new yaml file is missing this spring.devtools.remote.secret=mysecret.
Can you add that too? It is needed for livereload

File: build.gradle (hunk 1 above)
Comment (SabuhiYusif) [reply build.gradle:18]: I made it double quote since most of them is like that

File: build.gradle (hunk 2 above)
Comment (SabuhiYusif) [reply build.gradle:20]: This is postgres dependency for spring

**Pull request merged**